src/config_rc.ts

```diff
@@ -0,0 +1,114 @@
+import * as Controller from './controller'
+import * as Util from './utils'
+import * as Config from './config'
+import * as Native from './native_background'
+import * as Messaging from './messaging'
+import DEFAULTS from './config_defaults'
+import Logger from './logging'
+
+const RC_NAME = 'rc-text'
+const RC_LOC_NAME = 'rc-location'
+const RC_AUTOLOAD_FS_NAME = 'rc-autoload'
+const RC_DEFAULT = `
+; This is the default RC file.
+; Just like vim, comments start with ';'
```

Comment:
I think vim comments start with `"`, not `;`. Clojure comments do start with `;` though :)